magma/__init__.py

```diff
@@ -100,6 +100,6 @@ def set_mantle_target(t):
 from .syntax.coroutine import coroutine
 
 from magma.primitives import (LUT, Mux, mux, Register, get_slice, set_slice,
-                              slice, reduce, Memory)
+                              slice, Memory)
```

Comment:
Import is now covered by the existing `from .bits import *` 

![](https://media.giphy.com/media/DwIdasRkFKsMg/giphy.gif)

Reply:
😂